cleaner.py

```diff
@@ -0,0 +1,44 @@
+import nltk
+from nltk.corpus import stopwords
+from nltk.stem import PorterStemmer, WordNetLemmatizer
+from nltk.tokenize import sent_tokenize, word_tokenize
+from pyspark.sql import Row
+
+from spark.init_spark import init_spark
+
+nltk.download('punkt')
+nltk.download('stopwords')
+STOPWORDS_SET = set(stopwords.words('english'))
+  
+def main():
+    spark = init_spark()
+    rdd = spark.read.format('mongo').load().rdd
+    cleaned = clean_data(rdd)
+    cleaned.toDF().write.format("mongo").mode("overwrite").option("database", "aitaDB").option("collection", "cleanedinfo").save()
```

Comment:
Could we call the collection `cleaneddata` instead? Info feels like a strange word to use here imo 👀

Reply:
> Should we also remove punctuation? I don't think that's going to be useful. Also not sure about the importance of numbers for our algorithms of choice. Will they bring any extra semantic meaning? @tehblasian

@kwreen sure